app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala

```diff
@@ -412,12 +418,14 @@ class SubsidyController @Inject()(
 
   val claimTraderRefForm: Form[OptionalTraderRef] = Form(
     mapping(
-      "should-store-trader-ref" -> mandatory("should-claim-eori"),
```

Comment:
Might want to check this with @thebailers . We did some bug fixes around there

Reply:
yup I incorporated those in here :) any how I will rebase it once simon's changes are merged :)

Reply:
rebased finally , :) you can review now